spec/features/user_views_lists_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "User views all lists" do
+  scenario "successfully" do
+    create(:list, title: "Best Sci Fi")
+    book = create(:book, cover_image_url: 'http://placehold.it/140x100')
+    list_2 = create(:list, title: "Based on a true story")
+    list_2.books << book
+
+    visit lists_path
+
+    expect(page).to have_content("Best Sci Fi")
+    expect(page).to have_content("Based on a true story")
+    expect(page).to have_css("img[src*='http://placehold.it/140x100']")
```

Comment:
one way to decouple the image_url from the test would be to add an id and test for its presence:

```html
<%= image_tag book.cover_image_url, alt: book.title, id: "book-#{book.id}" %>
```

and in the spec

```ruby
expect(page).to have_css("#book-#{book.id}")
```

Or something similar with a class for the images. I'm fine with how it is now though, it's up to you guys.